compiler/plugins/target/ROCM/Dialect/ROCM/Transforms/test/apply_builtin_ukernel_pdl_patterns.mlir

```diff
@@ -1,10 +1,14 @@
-// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942 enable-tensor-ukernels=true}))' \
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-rocm-apply-builtin-pdl-patterns{targets=gfx942,gfx950 enable-tensor-ukernels=true}))' \
```

Comment:
I think this line would match all gfx942 or gfx950 patterns but I think a stronger check would be removing this line and everywhere you have `CHECK` now, replace it with both `GFX942` and `GFX950`. That would ensure that matching is happening for both targets (and not just one of them).